vtex-apps/modal-layout
# Add scroll content feature

#### What problem is this solving?

As title says

#### How should this be manually tested?

Go to the [workspace](https://modalscrollcontent--storecomponents.myvtex.com/) and click in the both rich texts:
Mobile support should be working too
![image](https://user-images.githubusercontent.com/8517023/71031669-9740c400-20f2-11ea-90e0-dbefe4d7b320.png)


#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
